### PR TITLE
Edited dockerfile names.

### PR DESCRIPTION
**Pull request by olliejth**
Resolved merge conflict and updated Dockerfile.

### CODE REVIEW

File: astronomy-pipeline/Dockerfile [hunk 1]
```diff
@@ -1,24 +1,8 @@
...
-
-COPY astronomy_load.py .
-COPY astronomy_extract.py .
-COPY astronomy_transform.py .
-COPY astronomy_extract_functions.py .
-COPY astronomy_transform_functions.py .
-COPY astronomy_lambda.py .
-
-CMD [ "astronomy_lambda.lambda_handler" ]
-=======
```
Comment (CalebMathers) [inline astronomy-pipeline/Dockerfile:21]: Consider yourself resolved, conflict!

File: astronomy-pipeline/Dockerfile [hunk 2]
```diff
@@ -28,8 +12,6 @@ COPY astronomy_transform.py .
 COPY astronomy_transform_functions.py .
 COPY astronomy_pipeline.py .
```
Comment (CalebMathers) [inline astronomy-pipeline/Dockerfile:13]: Nicer naming, in form with others

**Review by CalebMathers — APPROVED** [review]

File: astronomy-pipeline/Dockerfile (hunk 1 above)
Comment (AlexOBarnes) [reply astronomy-pipeline/Dockerfile:21]: This is very useful

File: astronomy-pipeline/Dockerfile (hunk 2 above)
Comment (AlexOBarnes) [reply astronomy-pipeline/Dockerfile:13]: Very nice naming

**Review by AlexOBarnes — APPROVED** [review]

**Pull request merged**